### PR TITLE
Changing 'C' to say clear

### PR DESCRIPTION
**Pull request by rafgeekee**
EDIT: Also changing class of the screen to `display`

### CODE REVIEW

File: calculator/index.html [hunk 1]
```diff
@@ -5,8 +5,8 @@
 <body>
 <div id="calculator">
 	<div class="top">
-		<span class="clear">C</span>
-		<div class="screen"></div>
+		<span class="clear">Clear</span>
```
Comment (william-eyre) [inline calculator/index.html:8]: Good clear class names

**Comment by beccahalford** [comment]
the commit message doesn't cover all the changes that you made, you changed the classes and modified the CSS file

File: calculator/index.html [hunk 2]
```diff
@@ -5,8 +5,8 @@
...
+		<div class="display"></div>
```
Comment (sonicintrusion) [inline calculator/index.html:9]: this change wasn't noted in the commit message

Comment (rafgeekee) [reply calculator/index.html:9]: Nice Spot!

**Comment by chrisprior1995** [comment]
This now shows clear in the show box instead of clearing the box

File: calculator/index.html (hunk 2 above)
Comment (rafgeekee) [reply calculator/index.html:9]: Resolved

**Review by chrisprior1995 — CHANGES_REQUESTED** [review]
Sort it out

**Comment by rafgeekee** [comment]
All erros fixed

**Comment by chrisprior1995** [comment]
Just to let you know this is not fixed try again have a nice day

**Review by RobertBMerriman — CHANGES_REQUESTED** [review]
https://github.com/essencious/git-advice/blob/2748b4fbbdc164dbe9b1dedfa2c188cd67ba1bae/calculator/main.js#L24

Change to 'Clear'

**Comment by rafgeekee** [comment]
I fixed it again

File: calculator/main.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ for(var i = 0; i < keys.length; i++) {
 		var inputVal = getInput();
 		var btnVal = this.innerHTML;
 
-		if(btnVal == 'C') {
+		if(btnVal == 'Clear') {
```
Comment (RobertBMerriman) [inline calculator/main.js:24]: Works now, nice

**Review by RobertBMerriman — APPROVED** [review]

**Review by MadeleineSmith — APPROVED** [review]
Clear button now works :D

**Review by CGDEV998 — APPROVED** [review]

**Review by pas-github — APPROVED** [review]

**Review by william-eyre — APPROVED** [review]
as far as i can tell, this should work

**Comment by chrisprior1995** [comment]
im liking it well done

**Pull request merged**